doc/whats-new.rst

```diff
@@ -23,6 +23,10 @@ v0.16.2 (unreleased)
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
+- :py:attr:`~core.accessor_dt.DatetimeAccessor.weekofyear` and :py:attr:`~core.accessor_dt.DatetimeAccessor.week`
+  have been deprecated. Use :py:attr:`~core.accessor_dt.DatetimeAccessor.isocalendar().week`
```

Comment:
unfortunately, this does not work: you can't reference the result of a call using `sphinx`. I'd suggest showing a code sample, e.g. ``arr.dt.isocalendar().week`` or ``ds.time.dt.isocalendar().week``

Reply:
thanks!